packages/design-system/src/components/Form/Field/Input/Input.Copy.tsx

```diff
@@ -31,15 +31,20 @@ const InputCopy = React.forwardRef(
 			<FieldGroup
 				label={label}
 				suffix={
-					!readOnly ? (
-						<Button.Icon
+					!readOnly || disabled ? (
```

Comment:
is this really wanted ? shouldn't it be `!disabled`
this change looks a but strange to me

Reply:
Either that or I need to pass `disabled` as a prop, thanks for spotting it!